src/quickgui/plugin/qgsquickpositionmarker.qml

```diff
@@ -0,0 +1,155 @@
+/***************************************************************************
+  qgsquickpositionmarker.qml
+  --------------------------------------
+  Date                 : Dec 2017
+  Copyright            : (C) 2017 by Peter Petrik
+  Email                : zilolv at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick 2.3
+import QtQuick.Controls 2.2
+import QtQml 2.2
+import QtGraphicalEffects 1.0
+import QgisQuick 0.1 as QgsQuick
+
+Item {
+  id: positionMarker
+  property int size: 48 * QgsQuick.Utils.dp
+
+  property QgsQuick.MapSettings mapSettings // required to be connected from parent!
+  property QgsQuick.PositionKit positionKit: QgsQuick.PositionKit {id: positionKit}
+
+  property var simulatePositionLongLatRad // use in debug mode to simulate movement around some GPS location
+  // longitude, latitude, and radius, all in degrees WSG84
+  // e.g. simulatePositionLongLatRad = [60, 10, 0.02]
+
+  property point screenPosition  // in pixels
+  property color baseColor: "darkblue"
+  property color unavailableColor: "gray"
+  property alias mapPosition: wgs84toMapCrs.projectedPosition // in map coordinates
+  property alias gpsPosition: positionKit.position // in WGS84 coordinates
```

Comment:
I'd rather refrain from using `GPS` explicitly in names, it's but one of the available systems.

Reply:
Do you have any preference for replacement?